sdk/rust-examples/huffman-encoding/README.md

```diff
@@ -0,0 +1,41 @@
+# A simple huffman encoding implementation with Veracruz freestanding execution engine
+
+
+This is a simple program that lets you implement the "Huffman Encoding Algorithm" which is a Lossless Compression Algorithm used to compresss Data. It was developed by David A. Huffman.  
+This directory is a simple program that takes a file called `huffman_example_input.txt` and compresses its data using the Huffman Encoding Algorithm, and then writes teh output into `encoded_output.txt`

```

Comment:
spelling: teh -> the

Reply:
fixed the typo. Thank you for finding this overlooked mistake!!